### PR TITLE
Update Next.js "Adding Tina" guide

### LINKED ISSUE #659: Update Tina+Next.js guide to add markdown plugin
see https://github.com/tinacms/tinacms.org/pull/650#issue-466093958

image

### PR DESCRIPTION
**Pull request by dwalkr**
includes instructions on installing the wysiwyg plugin

closes #659

### CODE REVIEW

File: content/guides/nextjs/adding-tina/creating-forms.md [hunk 1]
```diff
@@ -63,7 +62,7 @@ First, we'll need to import `useForm` and `usePlugin` from the `tinacms` package
 import { useForm, usePlugin } from 'tinacms'
 ```
 
-Now, just add the form to the `Post` component with the configuration we laid out previously:
+Then, just add the form to the `Post` component with the configuration we laid out previously:
```
Comment (DirtyF) [inline content/guides/nextjs/adding-tina/creating-forms.md:65]:
```suggestion
Then, add the form to the `Post` component with the configuration we laid out previously:
```

File: content/guides/nextjs/adding-tina/creating-forms.md [hunk 2]
```diff
@@ -108,12 +107,43 @@ In order to hook our form into the sidebar, we'll need to call `usePlugin` and p
...
-
 > **Why do we need to call usePlugin?**
 >
 > There are a few different ways to use forms: in the sidebar, in the global utility menu, and [inline](/docs/ui/inline-editing). How you plan to use the form will determine how you should set it up in the CMS.
 
+## Installing the WYSIWYG Plugin
+
+We're almost done! If you've made it this far, the form in your sidebar will look something like this:
+
+![](img/image.png)
```
Comment (DirtyF) [inline content/guides/nextjs/adding-tina/creating-forms.md:118]:
```suggestion
![](/img/image.png)
```

Image is not displaying on the preview

**Review by DirtyF — CHANGES_REQUESTED** [review]
The image doesn't display on https://tinacms-site-next-git-nextjs-adding-tina-guide-updates.tinacms.vercel.app/guides/nextjs/adding-tina/creating-forms/#installing-the-wysiwyg-plugin

File: content/guides/nextjs/adding-tina/creating-forms.md (hunk 2 above)
Comment (dwalkr) [reply content/guides/nextjs/adding-tina/creating-forms.md:118]: this must be a WYSIWYG bug

File: content/guides/nextjs/adding-tina/creating-forms.md (hunk 1 above)
Comment (dwalkr) [reply content/guides/nextjs/adding-tina/creating-forms.md]: The whole reason for this change was to get rid of that word and then I didn't

**Review by ncphillips — APPROVED** [review]

**Pull request merged**